src/dbt/kippmiami/models/fldoe/staging/stg_fldoe__fsa.sql

```diff
@@ -8,7 +8,7 @@ with
             scoreflag as score_flag,
 
             'Spring' as administration_window,
-            'FSA' as assessment_name,
+            'FSA' as test_name,
```

Comment:
@cbini Hey! Had to change this because it was failing the STAT dash and I need to refresh it to add a new filter. Hope that's OK! If not, please take points from Microsoft and not me :P

Reply:
fix it on the downstream table, that column needs to be consistent across all FLDOE files